web/src/main/webapp/xslt/services/pdf/metadata-fop.xsl

```diff
@@ -63,25 +66,23 @@
   <xsl:variable name="df">[Y0001]-[M01]-[D01]</xsl:variable>
 
 
-  <!-- ======================================================
-    FOP master configuration A4 with margins
-    -->
+  <!-- FOP master configuration A4 with margins -->
   <xsl:template name="fop-master">
     <fo:layout-master-set>
       <fo:simple-page-master master-name="simpleA4" page-height="29.7cm" page-width="21cm"
-                             margin-top="1.25cm" margin-bottom="1cm" margin-left="2cm"
+                             margin-top=".5cm" margin-bottom=".5cm" margin-left="2cm"
```

Comment:
To confirm the new margins doesn't cause issues when printing the report?

Reply:
Not for me.